### ISSUE TITLE
i'm not sure you can reference the hub with a short service name..

### ISSUE THREAD

**Issue by mdanidl** [Xin00163/terraform#1]
https://github.com/Xin00163/terraform/blob/a3cb6b3e4670b4dbce6490bbe413c27781e91691/fargate.tf#L210

**Comment by dom38** [comment]
Yeah we've seen that, the nodes don't register with the hub. Plan next is to make a hub app and spit out its IP, then make the node app and use that IP in the definition.

There's gotta be a better way though!

**Comment by mdanidl** [comment]
well, there are a few ways to do this, but the easiest one is probs this:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html#service-discovery-console

**Comment by mdanidl** [comment]
ahh, also there are some changes needed in the task definition. you'll need to link the containers together to allow comms. 

```{
  "containerDefinitions": [
    {
      "name": "wordpress",
      "links": [
        "mysql"                                      <==== this bit
      ],
      "image": "wordpress",
      ..........
    },
    {
      "name": "mysql",
      "image": "mysql",
      "essential": true
      ..........
    }
  ],
  "family": "hello_world"
}
```
